presto-native-execution/README.md

```diff
@@ -1,4 +1,16 @@
-# Prestissimo - C++ Presto worker implementation using Velox
+# Prestissimo - Presto Velox worker native execution
+
+> 💡 _**Notice:** Prestissimo is worker role implementation only and it still requires coordinator, Presto JAVA Server, to work._
+
+Prestissimo is a Presto Worker native execution query engine written using low-level programming languages like C++ and by design overcoming all of JAVA performance bottlenecks by utilizing modern CPU vector operations instructions sets like `avx`.
```

Comment:
There is a lot more to Prestissimo and Velox.
We don't have to explain Prestissimo here. Probably add a link to a blog when there is one.